.github/workflows/gradle.yml

```diff
@@ -19,6 +19,9 @@ jobs:
     - name: build and test cadc-vos
       run: cd cadc-vos && ../gradlew --info clean build javadoc install
       
+    - name: build and test cadc-vos-server-db
```

Comment:
this needs to be after cadc-vos-server in the build sequence since it implements the NodePersistence interface in there

Reply:
ah, you moved NodePersistence to the new module.... that's wrong. It needs to stay here so that cavern, for example, can implement it and not include cadc-vos-server-db in the classpath